scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -1748,6 +1822,12 @@ public DtlsConnectorConfig build() {
 			if (config.sniEnabled == null) {
 				config.sniEnabled = Boolean.FALSE;
 			}
+			if (config.useAntiReplayFilter == null) {
+				config.useAntiReplayFilter = true;
+			}
+			if (config.useWindowFilter == null) {
+				config.useWindowFilter = false;
+			}
```

Comment:
A detail, but it seems that setter does not allow a config where  `config.useAntiReplayFilter=true` and `config.useWindowFilter= true`

But if I set only userWindowsFilter to true, then the default value of useAntiReplayFilter will be true too.

Reply:
True, that's irritating.

DTLSConnector

```
this.useWindowFilter = config.useWindowFilter();
this.useFilter = config.useAntiReplayFilter() || useWindowFilter;

```

so it doesn't harm. But I will change the checks. I didn't wont "ambiguity"  in the configuration.